client/src/cl_cvarlist.cpp

```diff
@@ -180,7 +180,7 @@ CVAR_RANGE_FUNC_DECL(msgmidcolor, "5", "Color used for centered messages.",
 // ------------
 
 // Determines whether to draw the scores on intermission.
-CVAR(				wi_newintermission, "0", "Draw the scores on intermission",
+CVAR(				wi_newintermission, "0", "Use Odamex's intermission screen if there are 4 players or less on cooperative gamemodes.",
```

Comment:
Is there a convenient place to slot this into the menu system?

Reply:
It's already there for years... It just was left for many releases.
![image](https://user-images.githubusercontent.com/2938006/127725626-b2a671f0-b450-4196-8c3d-ab4e7513550a.png)